### ISSUE TITLE
Hex Editor multi-selection only updates last item

### ISSUE THREAD

**Issue by gregvanl** [microsoft/vscode-hexeditor#207]
Testing microsoft/vscode#114952

I'm not really sure the reason for multi-selection in the Hex Editor. You can't update multiple items at once as only the last highlighted item is updated.

**Comment by lramos15** [comment]
Looking for ideas on a better way to depict to the user which bytes are being used in the data inspector for the hex editor. Especially in the context of multi select. The data inspector will always take the leftmost selected byte and the next 7 bytes immediately following. This can be confusing if you don't know what's happening as the cell you have focused is not necessarily the cell the data inspector is getting its info from. I've included three screenshots one with left -> right select, another with right -> select, and lastly a random select one. I've highlighted in red what the data inspector is using to populate itself.

<img width="877" alt="l to r" src="https://user-images.githubusercontent.com/4544166/106005908-b4302080-6082-11eb-9cf8-c40cdae1681e.PNG">
<img width="885" alt="r to l" src="https://user-images.githubusercontent.com/4544166/106005924-b85c3e00-6082-11eb-9729-f6e8b1f4d97a.PNG">
<img width="884" alt="random select" src="https://user-images.githubusercontent.com/4544166/106005933-b98d6b00-6082-11eb-9834-e3c3c98d6d34.PNG">
